### PR TITLE
Prepare for b8 release, set changelog date and adjust pypi link.

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**

### CODE REVIEW

**Review by yunhaoling — DISMISSED** [review]
LGTM

**Comment by KieranBrantnerMagee** [comment]
Thanks for the migration guide/sample fixes, I hadn't been even thinking in that vein when proofreading but they're mighty good catches on all parts.

**Review by yunhaoling — APPROVED** [review]

File: eng/ignore-links.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 https://docs.microsoft.com/python/api/overview/azure/{{package_doc_id}}
-https://pypi.org/project/azure-servicebus/7.0.0b7/
```
Comment (weshaggard) [inline eng/ignore-links.txt:2]: 👍

**Pull request merged**